addons/mail/static/tests/web/attachment_box_tests.js

```diff
@@ -5,30 +5,6 @@ import { click, nextAnimationFrame, start, startServer } from "@mail/../tests/he
 
 QUnit.module("attachment box");
 
-QUnit.test("base empty rendering", async (assert) => {
-    const pyEnv = await startServer();
-    const partnerId = pyEnv["res.partner"].create({});
-    const views = {
-        "res.partner,false,form": `
-            <form>
-                <sheet></sheet>
-                <div class="oe_chatter">
-                    <field name="message_ids"  options="{'open_attachments': True}"/>
-                </div>
-            </form>
-        `,
-    };
-    const { openView } = await start({ serverData: { views } });
-    await openView({
-        res_id: partnerId,
-        res_model: "res.partner",
-        views: [[false, "form"]],
-    });
-    assert.containsOnce($, ".o-mail-AttachmentBox");
-    assert.containsOnce($, "button:contains('Attach files')");
-    assert.containsNone($, ".o-mail-Chatter .o-mail-AttachmentImage");
-});
-
```

Comment:
Removed this test because it makes no sense to show attachment box content when there are no attachments.